sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/OutputBinding/EventGridAttribute.cs

```diff
@@ -21,5 +21,10 @@ public sealed class EventGridAttribute : Attribute
         /// <summary>Gets or sets the Topic Key setting. You can find information on getting the Key for a topic here: https://docs.microsoft.com/en-us/azure/event-grid/custom-event-quickstart#send-an-event-to-your-topic </summary>
         [AppSetting]
         public string TopicKeySetting { get; set; }
+
+        /// <summary>
+        /// Gets or sets the app setting name that contains the Event Grid topic's connection string.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
Hmm this should probably have the AppSetting attribute as well.

Reply:
I don't think that would work. If I were just looking for a plain string value, no problem, but I need to access the configuration section and get child values out of it. If I apply the AppSetting attribute, the setting name would be replaced with the setting value before I get a chance to look at it, and since we expect the value to be an object then 'Connection' would just be set to `null`

Reply:
Ah I see - thanks for looking into that!